rott/rt_main.c

```diff
@@ -149,6 +154,15 @@ extern void RecordDemoQuery ( void );
 int main (int argc, char *argv[])
 {
     extern char *BATTMAPS, *ROTTMAPS;
+#ifdef __HAIKU__
```

Comment:
Why is this necessary? We only read/write files from/to `ApogeePath`, i.e. the result of `GetPrefDir()`, and only read files from the paths returned by `FindFileByName()`. Does any of these functions not work for you? If that's the case, I'd say we should go and fix it there instead of changing the directory to the executable just for Haiku.

Reply:
Ah sorry, this was a change applied to my sourceport fork because I was opening WADs from the executable's directory. In your sourceport it shouldn't be necessary. I'll remove it.